src/EditorFeatures/CSharpTest/CodeActions/InvertIf/InvertIfTests.Elseless.cs

```diff
@@ -0,0 +1,602 @@
+        switch (o)
+        {
+            case 1:
+                [||]if (c)
+                {
+                    f();
+                    f();
+                }
+
+                break;
+        }
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        switch (o)
+        {
+            case 1:
+                [||]if (!c)
+                {
+                    break;
+                }
+
+                f();
+                f();
+
+                break;
+        }
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        public async Task TestElseless09()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        switch (o)
+        {
+            case 1:
+                [||]if (c)
+                {
+                    if (c)
+                    {
+                        return 1;
+                    }
+                }
+
+                return 2;
+        }
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        switch (o)
+        {
+            case 1:
+                [||]if (!c)
+                {
+                    return 2;
+                }
+
+                if (c)
+                {
+                    return 1;
+                }
+
+                return 2;
+        }
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        public async Task TestElseless10()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        switch (o)
+        {
+            case 1:
+                if (c)
+                {
+                    [||]if (c)
+                    {
+                        return 1;
+                    }
+                }
+                return 2;
+        }
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        switch (o)
+        {
+            case 1:
+                if (c)
+                {
+                    [||]if (!c)
+                    {
+                    }
+                    else
+                    {
+                        return 1;
+                    }
+                }
+                return 2;
+        }
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        public async Task TestElseless11()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        switch (o)
+        {
+            case 1:
+                [||]if (c)
+                {
+                    f();
+                }
+                g();
+                g();
+                break;
+        }
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        switch (o)
+        {
+            case 1:
+                [||]if (!c)
+                {
+                }
+                else
+                {
+                    f();
+                }
+                g();
+                g();
+                break;
```

Comment:
i'm not sure how i feel about these.  but i like that we still do offer the feature...  but i have a feeling that having some of the inverts will end up undesirable.    Let's keep for now.

Reply:
When there's no simpler replacement, we use this style as the fallback. so we can always refactor to invert an elseless if, useful when you want to add code afterwards.

Reply:
(bydesign)